dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -438,7 +443,7 @@ cat "${CHE_DEPLOYMENT_FILE_PATH}" | \
     oc apply --force=true -f -
 else
   echo "[CHE] Deploying Che on OpenShift Container Platform (image ${CHE_IMAGE})"
-  DEFAULT_CHE_DEPLOYMENT_FILE_PATH=./che-spi-openshift.yml
+  DEFAULT_CHE_DEPLOYMENT_FILE_PATH=./che-che6-openshift.yml
```

Comment:
maybe we could rename file to be `che6-openshift.yml` instead of `che-che6-`

Reply:
:) yep